src/util/sequelize.ts

```diff
@@ -1,6 +1,27 @@
 import config from './config';
 import { Dialect, Sequelize } from 'sequelize';
 
+const options =
+  process.env.NODE_ENV === 'test'
+    ? {
+        host: config.dbHost,
+        port: Number(config.dbPort),
+        dialect: config.dbDialect as Dialect,
+        logging: false,
+        dialectOptions: {
+          ssl: {
+            require: true,
+            rejectUnauthorized: false,
```

Comment:
Can't connect to the heroku instance without this option.
See `rejectUnauthorized` option of node's [`TLSSocket`](https://nodejs.org/api/tls.html#tls_class_tls_tlssocket) (used by `node-pg` that is used by `sequelize`)